core/src/main/java/com/rackspace/com/papi/components/checker/util/DateUtils.java

```diff
@@ -41,12 +41,7 @@
 import java.lang.ref.SoftReference;
 import java.text.ParsePosition;
 import java.text.SimpleDateFormat;
-import java.util.Calendar;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.Locale;
-import java.util.Map;
-import java.util.TimeZone;
+import java.util.*;
```

Comment:
This is part of the IntelliJ IDEA settings. The default threshold for converting from individual import to package is five. Since there were six, it converted it to a package import.